### DISCUSSION TITLE
[Shader] SkSL, Support ?

### DISCUSSION THREAD

**Discussion by notour** [mono/SkiaSharp#1994] [Q&A]
Since version m81 the "Skia" base library support custom shader through SkSL and SkRuntimeEffect.
Does SkiaSharp (m88) support custom shader with SkSL ? if yes, how ? if not, it's in the pipe ?

Thanks

**Comment by mattleibow** [comment]
It does already. 😁 Check out the previews on nuget and use the SKRuntimeEffect type.
